lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -1399,6 +1406,9 @@ bool PotentialArchetype::addConformance(ProtocolDecl *proto,
     return false;
   }
 
+  // Bump the generation.
```

Comment:
If you're going to leave this comment on each, maybe it'd be a little nicer packaged into a `bumpGeneration` method on `GenericSignatureBuilder::Implementation`?